.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on: [push]
+
+env:
+  CI: true
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+        node-version: [12.x, 10.x]
+    steps:
+    - uses: actions/checkout@v2.1.0
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1.4.1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Cache node modules
+      uses: actions/cache@v1.1.2
+      env:
+        cache-name: cache-node-modules
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-node${{ runner.node-version }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+    - run: npm ci
+    - run: npm run build
+    - run: npm test
+    - run: npx prettier --check "src/**"
+    - run: npm audit --audit-level=moderate
```

Comment:
Should we run this in the `examples` dirs as well, you think?

Reply:
Given the examples aren't shipped with the code, I'm not sure it is necessary, but if it's easy to do and does not add noise that seems reasonable. I think I'd try it to see how noisy it turns out to be, only to disable it if the cost outwheights the benefits.

Reply:
Barely any noise: b6a94d0a437e67359a1e077defb61a4c92cf9b55

Reply:
Ah, but apparently it doesn't like the relative dependency. I'll remove it then.